### ISSUE TITLE
Not connecting

### ISSUE THREAD

**Issue by Vincen68** [hawkeye-stan/msfs-popout-panel-manager#124]
Hi

Firstly thank you for this great program. I am afraid it stopped working since the new update I believe. I unzipped the file and when I run PPM it appears I cannot connect to the sim no matter if I choose Fenix or FBW. The Start Pop Out button is greyed out and there is a red icon on the top left of the screen suggesting it si snot connected to MSFS. I could only think of 2 ways to troubleshoot it, once was to verify if simconnect is working and I believe it is working as FSUIPC is connecting to the sim and the second thing I could do is to repair the VC++ library, in case it was corrupted, but I still cannot connect to the sim. When in the sim, on a runway, PPM recognise the aircraft I have loaded (tried Fenix and FBW) but the icon on the top left of the window is red and the pop out button is greyed out. Not sure what else to do to troubleshoot this issue so any suggestion would be very welcome.

**Comment by Vincen68** [comment]
Sorry, forgot to add in the folder where the .exe file is located there is no error log

**Comment by Vincen68** [comment]
Apology, I found the error log in the document folder and it simply says
2024-07-31 17:03:35, 747 - Simconnector: SimConnect receive message exception - Acces is denied.

**Comment by hawkeye-stan** [comment]
Looks like something is blocking SimConnect from connecting to MSFS. Would you check if you have any antivirus installed that is blocking POPM since it may think updating POPM becomes a new app? If nothing is blocking POPM, I'm not sure what could be the cause since I haven't seen this particular "Access is denied" exception before and am not able to simulate it. You could try roll back to previous version 4.1.0 to see if you still have connection error. If so, something is definitely blocking SimConnect. To roll back, just download 4.1.0 zip file and overwrite current version.

**Comment by Vincen68** [comment]
Hi.

I was thinking about rolling back to the previous version but then I had some time today so I decided to do more troubleshooting.
I added POPM to the antivirus exception list (even though it was not necessary before). The firewall was already set to allow it so I did not touch the settings. I did several restarts of PC, MSFS, POPM after every step and also noticed an update for FSUIPC was due which I downloaded and installed. Finally, for good measure, I removed MSFS from running in admin mode (I had to do this previously as MSFS got stuck on the screen looking for updates for several minutes and someone suggested it was one way to resolve the issue). I now managed to get POPM working however I did not extensively test it as I did not have time to do so. I am leaning towards MSFS running in admin mode interfering with SimConnect (but I cannot be sure because FSUIPC did connect and so did LittleNavMap). Anyway, it appears one of the steps (or a combination of many of them) managed to get POPM connected to the sim. I will do more testing over the weekend but in the meantime thank you very much for your reply and if I still have issues I will roll back to the previous version.
